packages/fxa-auth-server/lib/routes/subscriptions/stripe.ts

```diff
@@ -468,6 +479,15 @@ export class StripeHandler {
       idempotencyKey,
     } = request.payload as Record<string, string>;
 
+    const planCurrency = (await this.stripeHelper.findPlanById(priceId))
+      .currency;
+    const paymentIntentCurrency = (
+      await this.stripeHelper.getPaymentIntent(paymentMethodId)
```

Comment:
This is a payment method ID, not a payment intent ID. For the payment method, we only have a country of origin attached.